about.html

```diff
@@ -151,10 +151,10 @@ <h3>Simple Budgeting</h3>
   <section class="partner" id="footer">
     <h3>Partner</h3> <hr>
     <ul>
-      <li>Bank-Central</li>
-      <li>Guinea-Games</li>
-      <li>freeCodeCamp</li>
+      <li>Google</li>
+      <li>Central-Banking</li>
       <li>Microverse</li>
+      <li>freeCodeCamp</li>
     </ul>
   </section>
 
```

Comment:
:red_circle: Please make sure the About Page footer has a dark background as it is in the given design.

| Your about page footer| Template About page Footer|
|:------------------------------:|:------------------------------------:|
|![stdesign](https://github.com/aradradev/easyBankTC/assets/26736582/212d409a-d540-47d4-86db-80779ffe90e2)|![about-footer](https://github.com/aradradev/easyBankTC/assets/26736582/ef985898-872b-45e6-ad33-c6834fa7f760)|